src/main/lens-proxy/lens-proxy.injectable.ts

```diff
@@ -27,6 +28,7 @@ const lensProxyInjectable = getInjectable({
     contentSecurityPolicy: di.inject(contentSecurityPolicyInjectable),
     emitAppEvent: di.inject(emitAppEventInjectable),
     logger: di.inject(loggerInjectable),
+    certificate: di.inject(lensProxyCertificateInjectable).get(),
```

Comment:
If `set()` always throws why not return `cert` directly without extra `.get()`?

Reply:
Because the renderer needs to fetch it async from the main (and the injection token aka interface is the same for both sides). I decided to _not_ use async/await here because it would have caused a big refactoring.